host/xtest/aes_gcm_counter_overflow_test_decrypt.h

```diff
@@ -19,7 +19,8 @@
 	.ctx_len = 40,
 	.tag = (const uint8_t []){ 0x15, 0x2a, 0x65, 0x04, 0x5f, 0xe6, 0x74, 0xf9, 0x76, 0x27, 0x42, 0x7a, 0xf5, 0xbe, 0x22, 0xda, },
 	.tag_len = 16,
-	.line = 77
+	.line = 6,
```

Comment:
Having the line number related to this generated header is nice. Thanks.

Reply:
welcome